gridt/package-lock.json

```diff
@@ -1083,168 +1083,68 @@
       }
     },
     "@angular/pwa": {
-      "version": "0.901.1",
```

Comment:
I recommend against updating the package-lock.json file in these small updates since that makes it hard to track when dependencies have updated.

Reply:
Yes, you can make a revert of a single file with `git checkout <commit_hash> -- <file>`